src/Exception/MaintenanceModeActiveHttpException.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\Exception;
+
+use Symfony\Component\HttpKernel\Exception\ServiceUnavailableHttpException;
+
+/**
+ * This exception is thrown when maintenance mode is active.
+ *
+ * @author Christian Schiffler <https://github.com/discordier>
+ */
+class MaintenanceModeActiveHttpException extends ServiceUnavailableHttpException
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function __construct(
+        // FIXME: should we initialize this to a sane default?
```

Comment:
Yes, that's exactly what we do :-) Then it will come back in an hour and everything will be fine. If it comes back in an hour and the site is still in maintenance mode it will come back in yet another hour. However, if we do not send a `Retry-After` header, the crawler itself has to decide whether it wants to crawl again or it just declares this page as completely broken and comes back in 30 days...who knows :)